src/main/java/org/apache/accumulo/testing/performance/tests/ConditionalMutationsPT.java

```diff
@@ -263,53 +231,41 @@ private static void runRandomizeBatchScanAndWriteTest(Environment env, String ta
     BatchWriter bw = env.getClient().createBatchWriter(tableName, new BatchWriterConfig());
     BatchScanner bs = env.getClient().createBatchScanner(tableName, Authorizations.EMPTY, 1);
 
-    boolean randomize = true;
-
-    randomizeBatchWriteAndScanTime(bw, bs, null, "0", reportBuilder, randomize);
+    randomizeBatchWriteAndScanTime(bw, bs, null);
 
     double rateSum = 0;
 
     for (int i = 1; i < 20; i++) {
-      rateSum += randomizeBatchWriteAndScanTime(bw, bs, (long) i, ((Integer) i).toString(),
-          reportBuilder, randomize);
+      rateSum += randomizeBatchWriteAndScanTime(bw, bs, (long) i);
     }
 
-    reportBuilder.result("avgRate", new Double(new DecimalFormat("#0.00").format(rateSum / 20)),
-        "RandomizeBatchScanAndWriteTest:  average rate (in seconds) to write and scan sequence 1-19");
-
-    // System.out.printf("rate avg : %6.2f \n", rateSum / 20);
-    // System.out.println("Flushing");
+    reportBuilder.result("avgRate: 1-19",
+        new Double(new DecimalFormat("#0.00").format(rateSum / 20)),
+        "RandomizeBatchScanAndWriteTest: average rate (in seconds) to write and scan sequence 1-19");
 
     env.getClient().tableOperations().flush(tableName, null, null, true);
 
     rateSum = 0;
     for (int i = 20; i < 40; i++) {
-      rateSum += randomizeBatchWriteAndScanTime(bw, bs, (long) i, ((Integer) i).toString(),
-          reportBuilder, randomize);
+      rateSum += randomizeBatchWriteAndScanTime(bw, bs, (long) i);
     }
 
-    reportBuilder.result("avgRate", new Double(new DecimalFormat("#0.00").format(rateSum / 20)),
-        "RandomizeBatchScanAndWriteTest:  average rate (in seconds) to write and scan sequence 20-39 post flush");
-
-    // System.out.printf("rate avg : %6.2f \n", rateSum / 20);
+    reportBuilder.result("avgRate: 20-39",
+        new Double(new DecimalFormat("#0.00").format(rateSum / 20)),
+        "RandomizeBatchScanAndWriteTest: average rate (in seconds) to write and scan sequence 20-39 post flush");
   }
 
-  private static double randomizeBatchWriteAndScanTime(BatchWriter bw, BatchScanner bs, Long seq,
-      String testRun, Report.Builder reportBuilder, boolean randomize) throws Exception {
+  private static double randomizeBatchWriteAndScanTime(BatchWriter bw, BatchScanner bs, Long seq)
```

Comment:
Should probably use lower-case `long` instead of `Long`, if `null` has no special meaning. Otherwise, this is creating unnecessary boxing/unboxing.

Reply:
I agree, I am not fully sure on the initial reasoning behind using `Long`. I made the change and replaced the null's with 0 to keep her code logic mostly the same.